.travis.yml

```diff
@@ -12,6 +12,10 @@ rust:
 os:
   - linux
   - osx
+addons:
```

Comment:
Hmm, we shouldn't be needing these changes to `.travis.yml`. Looks like I forgot to remove them in my branch...